libkbfs/bare_root_metadata_v3.go

```diff
@@ -88,16 +88,19 @@ type BareRootMetadataV3 struct {
 	// of writing to the given folder.
 	FinalizedInfo *tlf.HandleExtension `codec:"fi,omitempty"`
 
-	// The sequence number of the global Keybase Merkle tree at the
-	// time this update was created (from the writer's perspective).
-	// This field was added to V3 after it was live for a while, and
-	// older clients that don't know about this field yet might copy
-	// it into new updates via the unknown fields copier. Which means
-	// new MD updates might end up referring to older Merkle roots.
-	// That's ok since this is just a hint anyway, and shouldn't be
-	// fully trusted when checking MD updates against the Merkle tree.
+	// DEPRECATED -- see KBMerkleRoot.Seqno instead.
```

Comment:
I guess alternatively we could also just add `keybase1.HashMeta` for on-the-wire message, because `keybase1.MerkleRootV2` is essentially `keybase1.Seqno` plus `keybase1.HashMeta`. The only difference is `keybase1.Seqno` is `int64`, while `libkbfs.Seqno` is `uint64`. Not sure if it makes any difference for msgpack tho.